CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to Fetch.ai ledger
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the [Fetch.ai ledger repository](https://github.com/fetchai/ledger) on GitHub. We strongly encourage you to follow them as closely as possible; if you are dealing with a case not specified here, use your best judgment and feel free to propose changes to the document in a pull request.
+
+
+## Bug reports
```

Comment:
I think it might be nice to include an issue template covering this section also:

https://help.github.com/en/github/building-a-strong-community/creating-issue-templates-for-your-repository

In addition it might be nice to have an pull request template also

Reply:
From discussion - sounds like we will make this a new PR